apps/greenhouse/src/Shell.test.jsx

```diff
@@ -9,18 +9,11 @@ import { render } from "@testing-library/react"
 // https://reactjsexample.com/an-extension-of-dom-testing-library-to-provide-hooks-into-the-shadow-dom/
 import Shell from "./Shell"
 import Auth from "./components/Auth"
-import StoreProvider from "./components/StoreProvider"
 import { vi } from "vitest"
 
-vi.mock("@cloudoperators/juno-communicator")
 vi.mock("./components/Auth")
 
 test("renders app", () => {
-  render(
-    <StoreProvider>
-      <Shell />
-    </StoreProvider>
-  )
-
+  render(<Shell mockAuth={true} />)
```

Comment:
How should it work? The Shell component does not require or handle any props.

Reply:
You're right, thank you! It might be a leftover from when the tests were being fixed